packages/aragon-cli/package.json

```diff
@@ -89,11 +89,11 @@
   "devDependencies": {
     "@aragon/apps-agent": "^2.0.0",
     "@aragon/apps-finance": "^3.0.0",
+    "@aragon/apps-payroll": "^1.0.0",
```

Comment:
Avoid unnecessary changes in other sections of the package.json to prevent git conflicts

Reply:
Agree that we should avoid changes unrelated to the PR. I left it there because it will probably conflict with `develop` if I remove it.